cparser/Elab.ml

```diff
@@ -1403,10 +1434,10 @@ let elab_expr vararg loc env a =
             | TUnion(id, attrs) ->
                 (wrap Env.find_union_member loc env (id, fieldname), attrs)
             | _ ->
-                error "left-hand side of '->' is not a pointer to a struct or union"
+                error "member reference base type %a is not a struct or union" (print_typ env) b1.etyp
             end
         | _ ->
-            error "left-hand side of '->' is not a pointer " in
+            error "member reference type %a is not a pointer" (print_typ env) b1.etyp in
```

Comment:
Again, I find `left-hand side of '->'` more descriptive than `member reference`.